python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                             'R0913': 'The function has too many parameters (%s out of a default limit '
+                                      'of %s). You should try to reduce the complexity of the function '
+                                      'by splitting up it, or combining related objects as a single one.',
+                             'R0914': 'The function has too many local variables (%s out of a default limit '
+                                      'of /%s).',
+                             'R0915': 'The function or method has too many statements (%s out of a default limit '
+                                      'of /%s). It should be split up into smaller functions or methods.',
+                             'R0916': 'The if statement contains too many boolean expressions '
+                                      '(%s out of a default limit of %s). You should see if you can reduce '
+                                      'the amount of expressions, or split it into branches.',
+                             }
+    },
+    'pylint.checkers.classes': {
+        'ClassChecker': {'E0203': 'You can\'t use class member %r prior to it\'s assignment '
+                                  'on line %s.',
+                         'E0211': 'Every class method requires at least one parameter, '
+                                  'which we conventionally name \'self\'.',
+                         'E0213': 'The first parameter of a method should always be called \'self\'. '
+                                  'This is a strongly adhered to convention.',
+                         'E0241': 'The class %r should not inherit from the same class multiple times.',
+                         'R0201': 'Method does not make use of \'self\', so you should rewrite the '
+                                  'method as a function and move it outside the class.',
+                         'R0205': 'Class %r inherits from \'object\', which can be omitted in Python3.',
+                         'W0211': 'Static method %r shouldn\'t contain \'self\' as the first parameter, '
+                                  'since this suggests we are expecting a class instance as the first argument.',
+                         'W0212': 'Since %s starts with an underscore, it should be considered '
+                                  '"private", and not accessed outside of the class in which it is '
+                                  'defined.',
+                         'W0221': 'Method must take the same number of arguments as %s %r method.',
+                         'W0222': 'The method parameters must have the same name, order, and default '
+                                  'arguments as %s %r method.',
+                         'W0233': 'The abstract method %r in class %r must be overridden within a '
+                                  'concrete class.',
+                         'W0231': 'Child class should call the \'__init__\' method from it\'s base class %r.',
```

Comment:
child class -> subclass __init__ method

it's -> its

base class -> superclass

Reply:
similar changes to W0233 on the next line